### DISCUSSION TITLE
Error Module Parse Failed

### DISCUSSION THREAD

**Discussion by CarlosRicoSaluta** [matteobruni/tsparticles#5278] [Q&A]
Hi, I am trying install tsparticles in React JS, I follow the readme but get the error:

./node_modules/@tsparticles/engine/browser/Utils/ColorUtils.js 107:29
Module parse failed: Unexpected token (107:29)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
| }
| export function stringToAlpha(input) {
>   return stringToRgba(input)?.a;
| }
| export function stringToRgb(input) {

I had used the previous version previously and it worked for me, but now I also get the same error with that version

**Comment by matteobruni** [comment]
It is something in your transpiler that is missing, have you checked something like this?

https://medium.com/@duvvurukishore/how-i-resolved-you-may-need-additional-loader-to-handle-the-result-of-these-loaders-error-919a2e0356a0

**Reply by CarlosRicoSaluta** [reply]
Yes, I checked that and try anythings but nothing work

**Reply by matteobruni** [reply]
I don't know, it depends on your transpiler and transpiler configuration.
